rmgpy/rmg/main.py

```diff
@@ -1193,7 +1327,6 @@ def readMeaningfulLineJava(self, f):
         return line
     
 ################################################################################
-
```

Comment:
It may make the plain code look cleaner, which is an improvement. Unfortunately a consequence is that modifying whitespace mixed in with other changes makes understanding the changes (git blame), (git show() or (git log --follow) more messy. 

If you'd like to improve some formatting, I think it would be best practices to improve the formatting of a whole file (or region) and make the formatting improvement a separate commit (and possibly pull request).